dirty_cat/_minhash_encoder.py

```diff
@@ -108,18 +107,18 @@ def minhash(
 
         Returns
         -------
-        array, shape (n_components, )
+        ndarray of shape (n_components, )
             The encoded string.
         """
-        min_hashes = np.ones(n_components) * np.infty
+        min_hashes = np.ones(self.n_components) * np.infty
```

Comment:
As mentioned [here](https://numpy.org/devdocs/reference/constants.html#numpy.infty), should be using `np.inf`.
```suggestion
        min_hashes = np.ones(self.n_components) * np.inf
```